Update Google Cloud docs with Docker recipe (#2935)
withastro/docs: src/content/docs/en/guides/deploy/google-cloud.mdx

3. Create a new service.

#### Create Dockerfile & Build the Container

> Before you can deploy your Astro site to Cloud Run, you need to create a Dockerfile that will be used to build the container. You can find more [documentation on how to use Docker with Astro in our Docker Recipe](/en/recipes/docker/#creating-a-dockerfile).

Reviewer: ```suggestion
Before you can deploy your Astro site to Cloud Run, you need to create a Dockerfile that will be used to build the container. Find more information about [how to use Docker with Astro](/en/recipes/docker/#creating-a-dockerfile) in our recipe section.
```
I love a PR that deletes so many lines!  :partying_face: I'm just tightening up the call to action, then merging! Bye bye, all these lines!

> Before you can deploy your Astro site to Cloud Run, you need to create a Dockerfile that will be used to build the container. You can find more [documentation on how to use Docker with Astro in our Docker Recipe](</en/recipes/docker/#creating-a-dockerfile>).

Reviewer 2: I'm wondering why there are arrows in this link. Just to be sure we don't fall into some very specific error in the future, I'd remove this.
```suggestion
Before you can deploy your Astro site to Cloud Run, you need to create a Dockerfile that will be used to build the container. You can find more [documentation on how to use Docker with Astro in our Docker Recipe](/en/recipes/docker/#creating-a-dockerfile).
```

Author: 😱 I always do this! seems like it's not needed at all 😅 It does work tho!
